src/main/webapp/view/video/video.js

```diff
@@ -12,15 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import Synchronizer from '../../synchronizer.js';
 import {secondsToMilliseconds} from '../../timestamps.js';
 
 const SCRIPT = 'script';
 
 /** Initializes and stores video player information. */
 export default class Video {
+  #lecture;
+  #synchronizer;
+
+  /** Creates a new `Synchronizer` linked to `this`. */
```

Comment:
I find it weird the constructor only mentions creating a new synchronizer but not a new Video instance. I think `lecture` should be mentioned as well.